public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -188,7 +188,7 @@ export const DashNav = React.memo<Props>((props) => {
     if (dashboard.meta.publicDashboardEnabled) {
       // TODO: This will be replaced with the new badge component. Color is required but gets override by css
       buttons.push(
-        <Badge color="blue" text="Public" className={publicBadgeStyle} data-testid={selectors.publicDashboardTag} />
+        <Badge color="blue" text="Public" key="button-badge" className={publicBadgeStyle} data-testid={selectors.publicDashboardTag} />
```

Comment:
```suggestion
        <Badge color="blue" text="Public" key="public-dashboard-button-badge" className={publicBadgeStyle} data-testid={selectors.publicDashboardTag} />
```
Super small nit, just to not be too generic. Sorry for being picky here 😅

Reply:
Makes total sense!